hackathon_submission/ListBlobVersioning.ipynb

```diff
@@ -0,0 +1,287 @@
+{
```

Comment:
Can we remove this file 😬

Reply:
We could remove this one. However, I deliberately put this Jupyter Notebook file for viewing the results obtained and easy verification of the code.

Reply:
BTW don’t forget to rotate your account key else you’ll be asked to rotate in an email 😁

Reply:
Sure!! Thanks 😃